src/main/scala/com/microsoft/hyperspace/index/rules/FilterIndexRule.scala

```diff
@@ -147,12 +97,16 @@ object FilterIndexRule
       outputColumns: Seq[String],
       filterColumns: Seq[String],
       fsRelation: HadoopFsRelation): Seq[IndexLogEntry] = {
+    val indexManager = Hyperspace
+      .getContext(spark)
+      .indexCollectionManager
```

Comment:
I am just curious, is there a reason to move this code snippet outside of `Some` block. We dont use IndexManager for the `None` case.

Reply:
I'll revert this in #165. Thanks :)